algorithm-prototyping/src/main/resources/scripts/generate_distance_matrix.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+# Read buildings.json and store data as a dictionary
+with open('buildings.json', 'r') as file:
+    data = json.load(file)
+
+# Initialize output dictionary with buildings array and distance matrix
+distanceData = {
+    'distance_matrix': {}

```

Comment:
Remove `distance_matrix` key. See format for `distance_matrix.json`:

```json
{
  "Park Hall": {
    "Park Hall": 0.0,
    "Boyd": 10.0,
    ...
  },
  "Boyd": {
    "Park Hall": 10.0,
    "Boyd": 0.0,
    ...
  },
  ...
}
```

Make sure the rest of the code reflects this change.